commanderbot_ext/ping/ping_cog.py

```diff
@@ -0,0 +1,13 @@
+import discord
+from commanderbot_lib.logging import Logger, get_clogger
+from discord.ext.commands import Bot, Cog, Context, command
+
+
+class PingCog(Cog):
+    def __init__(self, bot: Bot):
+        self.bot: Bot = bot
+        self._log: Logger = get_clogger(self)
+
+    @command(name="ping")
+    async def cmd_ping(self, ctx: discord.Message):
+        await ctx.send(f"Pong! Latency: {round(self.bot.latency, 2)}ms")
```

Comment:
Side note: I'm on the fence about whether anything else should be in a simple ping command, but given that this is a [supported property](https://discordpy.readthedocs.io/en/latest/api.html#discord.Client.latency) from the discord.py API I think it's fine.

Reply:
Fixed in latest commit (`ms` to `s`)